src/http/ngx_http_lua_headers.c

```diff
@@ -1166,7 +1166,8 @@ ngx_http_lua_ffi_req_get_headers(ngx_http_request_t *r,
 int
 ngx_http_lua_ffi_set_resp_header(ngx_http_request_t *r, const u_char *key_data,
     size_t key_len, int is_nil, const u_char *sval, size_t sval_len,
-    ngx_http_lua_ffi_str_t *mvals, size_t mvals_len, char **errmsg)
+    ngx_http_lua_ffi_str_t *mvals, size_t mvals_len, unsigned override,
```

Comment:
Can we use `int` instead of `unsigned` for `override` here? We usually use `int` for C boolean types (unless it is a bit fields in a struct).

Reply:
@agentzh 
Changed in https://github.com/openresty/lua-nginx-module/pull/1289.